### PR TITLE
Small HTML Cart/Coupons Fixes 

### PR DESCRIPTION
**Pull request by antonioperic**
Small HTML Cart/Coupons Fixes - adding and removing coupon

### CODE REVIEW

File: src/Sylius/Bundle/WebBundle/Resources/views/Frontend/Cart/summary.html.twig [hunk 1]
```diff
@@ -78,12 +92,19 @@
...
+    <div id="sylius-cart-coupons" class="collection-container"
+         data-prototype="{{ _self.coupon_item(form.promotionCoupons.vars.prototype)|e }}">
+        {% for promotionCoupon in form.promotionCoupons %}
+            {{ _self.coupon_item(promotionCoupon) }}
+        {% endfor %}
     </div>
+
     <br>
-    <a href="#" class="btn btn-success btn-block" data-collection-button="add" data-prototype="sylius-cart-coupons" data-collection="sylius-cart-coupons-collection">
+    <a href="#" class="btn btn-success btn-block" data-collection-button="add"
```
Comment (aRn0D) [inline src/Sylius/Bundle/WebBundle/Resources/views/Frontend/Cart/summary.html.twig]: It is a collection, you should use the Form Collection Extension which provide default template and JS/

**Comment by antonioperic** [comment]
@aRn0D will rowork this during next few days

File: src/Sylius/Bundle/WebBundle/Resources/views/Frontend/Cart/summary.html.twig (hunk 1 above)
Comment (antonioperic) [inline src/Sylius/Bundle/WebBundle/Resources/views/Frontend/Cart/summary.html.twig]: We have two form-collections (resource + web) is that any mistake, or everything is ok.
Which one should be used on frontend

**Comment by antonioperic** [comment]
@aRn0D I will need some "hint" or help, I included resource form collection JS, but onClick event is never executed. HTML code seems ok with all scenery things.

    <div class="col-md-12 well well-sm">
    <div class="form-group"><label class="col-lg-2 control-label">If you have a promotion coupon, enter it here</label><div class="col-lg-10 clearfix"><div data-prototype="    &lt;div data-form-collection=&quot;item&quot;
             data-form-collection-index=&quot;__name__&quot;
             class=&quot;collection-item&quot;&gt;&lt;div class=&quot;collection-box&quot;&gt;&lt;p class=&quot;text-right&quot;&gt;&lt;a href=&quot;#&quot; data-form-collection=&quot;delete&quot; class=&quot;btn btn-danger&quot;&gt;&lt;i class=&quot;glyphicon glyphicon-remove collection-button-remove&quot;&gt;&lt;/i&gt;
                            Delete
                        &lt;/a&gt;&lt;/p&gt;&lt;input type=&quot;text&quot; id=&quot;sylius_cart_promotionCoupons___name__&quot; name=&quot;sylius_cart[promotionCoupons][__name__]&quot; required=&quot;required&quot; class=&quot; form-control&quot; /&gt;&lt;/div&gt;&lt;/div&gt;" class=" controls collection-widget" id="sylius_cart_promotionCoupons" data-form-type="collection"><div class="row collection-list" data-form-collection="list"><div class="collection-item" data-form-collection-index="0" data-form-collection="item"><div class="collection-box"><p class="text-right"><a class="btn btn-danger" data-form-collection="delete" href="#"><i class="glyphicon glyphicon-remove collection-button-remove"></i>
                            Delete
                        </a></p><input type="text" class=" form-control" name="sylius_cart[promotionCoupons][0]" id="sylius_cart_promotionCoupons_0"></div></div></div><a data-form-collection="add" class="btn btn-success btn-block" href="#"><i class="glyphicon glyphicon-plus"></i>
                    Add
                </a></div></div></div>
</div>

**Comment by antonioperic** [comment]
ping @aRn0D

**Comment by aRn0D** [comment]
@antonioperic Did you dump your assets? It seems correct and you should have a to the build too ;)

**Comment by antonioperic** [comment]
@aRn0D yes, dumped, cleared cache, everything...doesn't work. Any idea? Is seems that doesn't catch any click event

**Comment by pjedrzejewski** [comment]
We will take care of this in new theme, let's not waste time on this old piece of html. :)

**Comment by antonioperic** [comment]
@pjedrzejewski no prob. Is there any released date for new theme

**Comment by pjedrzejewski** [comment]
@antonioperic Soon. (tm) :(

**Pull request closed without merging**